orttraining/orttraining/python/training/orttrainer.py

```diff
@@ -910,7 +910,7 @@ def state_dict(self, pytorch_format=False):
                 type: dict,
                 schema:
                 {
-                    "fp32":
+                    "full_precision":
```

Comment:
did you update design doc to reflect this change?

Reply:
updating it now.